cmd/costmodel/main.go

```diff
@@ -1,29 +1,22 @@
 package main
 
 import (
-	"net/http"
+	"flag"
+	"os"
 
-	"github.com/julienschmidt/httprouter"
-	"github.com/kubecost/cost-model/pkg/costmodel"
-	"github.com/kubecost/cost-model/pkg/errors"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
-	"github.com/rs/cors"
+	"github.com/kubecost/cost-model/pkg/cmd"
 	"k8s.io/klog"
 )
 
-func Healthz(w http.ResponseWriter, _ *http.Request, _ httprouter.Params) {
-	w.WriteHeader(200)
-	w.Header().Set("Content-Length", "0")
-	w.Header().Set("Content-Type", "text/plain")
-}
-
 func main() {
-	a := costmodel.Initialize()
+	klog.InitFlags(nil)
+	flag.Set("v", "3")
```

Comment:
I know this is pulled from legacy, but I'm curious if hardcoding the log level is really a good idea.

Reply:
Yeah, I really like the simplicity of the klog library, but there are some weird things happening with this. I'm not sure if passing `-v` to the application overrides this or not. If that's the case, I'm fine with leaving the default at 3 (it looks like the default inside klog is 0).